src/Chapter1.hs

```diff
@@ -652,8 +668,11 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = (first (abs n))
+  where
+    first :: Int -> Int
+    first x = if (div x 10) > 0 then first (div x 10) else x
```

Comment:
Awesome implementation! Absolutely correct that you do `abs` in the beginning only once, so you don't need to do this in the helper recursive function.